notebooks/peerscout/peerscout-recommend-senior-editors.ipynb

```diff
@@ -180,39 +190,41 @@
     "    ),\n",
     "    editor_tf_idf\n",
     ")\n",
-    "print(keyword_similarity.max())\n",
-    "keyword_similarity"
-   ],
-   "outputs": [],
-   "metadata": {}
+    "print(keyword_similarity.max())"
+   ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "source": [
-    "weighted_keyword_valid_model.predict_ranking(\n",
-    "    manuscript_version_for_recommendation_df['extracted_keywords'][:1],\n",
-    ").proba_matrix"
-   ],
+   "metadata": {},
    "outputs": [],
-   "metadata": {}
+   "source": [
+    "# weighted_keyword_valid_model.predict_ranking(\n",
```

Comment:
did this cause an issue?

Reply:
no it didn't, it was just a very long log. We have removed this part for reviewing editors notebook before so I have just removed it here too.